datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -49,17 +50,35 @@ def agent_command(self):
             get_agent_exe(self.agent_version)
         )
 
-    def run_check(self, capture=False, rate=False):
-        command = '{} check {}{}'.format(
-            self.agent_command,
+    def exec_command(self, interactive=False):
+        return 'docker exec{} {}'.format(
+            ' -it' if interactive else '', self.container_name
+        )
+
+    def run_check(self, interactive=False, capture=False, rate=False):
+        command = '{} {} check {}{}'.format(
```

Comment:
Let's document what `command` can be with an example here to make it easier to understand